components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -692,6 +816,15 @@ public static Boolean isReCaptchaEnabled() {
         return CaptchaDataHolder.getInstance().isReCaptchaEnabled();
     }
 
+    /**
+     * Get the ReCaptcha Type.
+     * @return ReCaptcha Type as a String.
+     */
+    public static String reCaptchaType() {
```

Comment:
shall we change the method name to `getreCaptchaType()` to be consist with others?

Reply:
I actually put it as `reCaptchaType` to be consistent with the other methods in that class.
There are get methods named `reCaptchaSiteKey`, `reCaptchaAPIURL`, etc in that class.